oci-devops-cicd-helm-ci/helm-deployment/helm-deployment.md

```diff
@@ -0,0 +1,226 @@
+# Deployment of helm chart with valid provenance and integrity checks.
+
+## Introduction
+
+In this lab, you will be running the OCI DevOps pipeline to deploy a packaged helm chart with validation of its origin and signature.
+
+
+Estimated time: 20 minutes
+
+### Objectives
+
+In this lab, as a developer or SRE,
+
+* Create necessary keys for helm chart verification
+* Run a build and deployment pipeline with a helm chart.
+* Validate the helm deployment
+
+## Task 1: GPG Key Setup (For Helm Signing and Verification).
+
+1. We are using GnuPG (gpg) to set up the key for signing and verifying the helm chart. Read more about gpg [here](https://gnupg.org/).
+
+1. We are going to install gpg and the instruction here is related to a Linux machine. You may change the procedure accordingly - https://www.gnupg.org/howtos/card-howto/en/ch02.html.
+
+
+1. Follow the below instructions and download the stable version of gpg onto your workstation.
+
+```java
```

Comment:
@RahulMR42 Add copy button code blocks to the code and indent them with 4 spaces from begin of line. For more details, you can refer the guide - https://oracle-livelabs.github.io/common/sample-livelabs-templates/create-labs/labs/workshops/livelabs/?lab=4-labs-markdown-develop-content#Task12:CodeSnippets

Reply:
Done